server/test-enhanced-validation.js

```
@@ -0,0 +1,486 @@
+        }
+      }
+    };
+
+    try {
+      const response = await this.sendMCPRequest(request);
+      
+      if (response.error && response.error.code === -32602) {
+        console.log('✓ Correctly validates node existence');
+        console.log(`  Error message: ${response.error.message}`);
+        if (response.error.data && response.error.data.examples) {
+          console.log(`  Available nodes: ${JSON.stringify(response.error.data.examples)}`);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)